SDWebImage/SDWebImageManager.m

```diff
@@ -253,12 +245,6 @@ - (SDWebImageCombinedOperation *)loadImageWithURL:(nullable NSURL *)url
                     }
                     
                     BOOL cacheOnDisk = !(options & SDWebImageCacheMemoryOnly);
-                    
```

Comment:
This will be refactored using the custom cache key filter feature. So we pass the cache key filter to downloader, downloader use that to calculate cache key (or string from URL). We do not need this hack logic.